frontend/src/components/floating-menus/DialogModal.vue

```diff
@@ -90,5 +90,11 @@ export default defineComponent({
 			this.dialog.dismissDialog();
 		},
 	},
+	mounted() {
+		// Focus the first button in the popup
+		const element = this.$el as Element | null;
+		const emphasizedOrFirstButton = (element?.querySelector("button[data-emphasized='true']") as HTMLButtonElement | null) || element?.querySelector("button");
```

Comment:
That might be how we set it, yes, but for defensive programming (and general simplicity) it's cleaner if we test for the existence of the attribute rather than its `true` value. I believe it's (at least conceptually) something like `:data-emphasized="emphasized || null"` and Vue treats null as not-to-be-included-as-an-attribute-at-all. Hopefully that explains the reasoning!